### ISSUE TITLE
Have newrelic-ruby-agent-bot notify us about its PR behavior

### ISSUE THREAD

**Issue by kaylareopelle** [newrelic/newrelic-ruby-agent#1920]
### Description
When we release a new version and the newrelic-ruby-agent bot makes a PR to the docs-website with our release note changes, we want to know the PR was successfully opened and if any comments are made/need to be addressed so that we don't leave this hanging.

We get notifications for other PRs we work on in the #ruby-agent-notifications channel. Perhaps we can funnel the PR notifications into that channel.

### Acceptance Criteria
- [ ] Team is notified in Slack during the PR lifecycle (open, comment, merge, approve, etc.) for the newrelic-ruby-agent bot. 

### Design Consideration/Limitations
Currently, GitHub emails are sent to all members of the ruby-agent@newrelic.com email, so we can see the behavior through e-mail until we can get to this ticket. **Should we continue to use the email notifications once we have Slack notifications?** 

### Dependencies
We may need to notify the docs team before starting this work so they aren't confused/concerned by test PRs if we need to make them to meet the acceptance criteria 

### Additional context
Related to https://github.com/newrelic/newrelic-ruby-agent/issues/1919 

### Estimates
S

**Comment by fallwith** [comment]
Beyond the existing two known issues we have with notifications...

- Repo or org must have a notifications app installed
- An individual Slack user set up the subscription; complicating off-boarding

...this particular request presents us with the additional challenge of wanting to be notified of all repo activity but only for a single GitHub user.

The [integration](https://github.com/integrations/slack) seems to only offer user ("actor") filtration for workflows, but not other activity. There is a [now-4-year-old issue](https://github.com/integrations/slack/issues/878) asking for user filtration functionality, but there is no indication if the request has been considered.

Closing as won't-fix, given the original intention of this issue to use a quick notifications subscription approach. If we find ourselves still needing this type of notification in future, we can create a new issue later that calls for the investigation of other avenues, such as GitHub Actions - given that we control the repo in question and can run them.
